integration-tests/src/test/java/com/example/IntegrationTest.java

```diff
@@ -107,6 +107,20 @@ public void bindElementsIntoSetWrongReturn() {
     }
   }
 
+  @Test
+  @IgnoreCodegen
+  public void bindElementsIntoSetGenericWrongReturn() {
+    try {
+      backend.create(BindsElementsIntoSetGenericWrongReturn.class);
+      fail();
+    } catch (IllegalArgumentException e) {
+      assertThat(e)
+          .hasMessageThat()
+          .isEqualTo(
+              "@Binds methods must return a primitive, an array, a type variable, or a declared type. Found java.util.Set<? extends java.lang.String>.");
```

Comment:
Can you break this string at or around 100 chars?

Reply:
Formatted!